agents-for-bedrock/use-case-examples/hr-assistant/agent/knowledge-base-assets/pay-february.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
is this data generated?

Reply:
Yeah we synthetically generated the data.